docs/web3-eth-accounts-api.rst

```diff
@@ -250,4 +250,679 @@ Example
         result: "0xa1de988600a42c4b4ab089b619297c17d53cffae5d5120d82d8a92d0bb3b78f2"
     }
 
+------------------------------------------------------------------------------
+
+.. _sign:
+
+web3/eth/accounts/sign
+======================
+
+----
+POST
+----
+
+.. code-block:: javascript
+
+    http://localhost:3000/web3/eth/accounts/sign
+
+Signs arbitrary data. This data is before UTF-8 HEX decoded and enveloped as follows: "\x19Ethereum Signed Message:\n" + message.length + message.
```

Comment:
"x19Ethereum Signed Message:n" What does this mean?

Reply:
I copied it from web3.js docs.
https://web3js.readthedocs.io/en/v1.2.1/web3-eth-accounts.html#id16